reducers/heroes.js

```diff
@@ -0,0 +1,18 @@
+import { heroes } from 'dotaconstants';
+import { API_HOST } from './../yasp.config';
+
+// lazy evaluation
+// add API_HOST to hero img path so components
+// don't need to know API_HOST
+const transformHeroImagePath = (heroes) => {
+  const heroList = {};
+  Object.keys(heroes).forEach((key) => {
+    heroList[key] = { ...heroes[key] };
+    heroList[key].img = API_HOST + heroList[key].img;
+  });
+
```

Comment:
thank you to pointing out the selector pattern example. give me some time to refactoring my code :)